shared/common-adapters/index.tsx

```diff
@@ -55,6 +55,7 @@ export {default as LabeledInput} from './labeled-input'
 export {default as List} from './list'
 export {default as List2} from './list2'
 export {default as LoadingLine} from './loading-line'
+export {default as LocationMap} from './location-map'
```

Comment:
why is this common?

Reply:
I use it in two places, and thought it had potential to be used in other spots as well. Seems similar to QR scanner, which is in there.